src/main/java/teammates/ui/controller/StudentHomePageAction.java

```diff
@@ -26,9 +26,14 @@ public ActionResult execute() throws EntityDoesNotExistException {
         String recentlyJoinedCourseId = getRequestParamValue(Const.ParamsNames.CHECK_PERSISTENCE_COURSE);        
         
         try {
-            data.courses = logic.getCourseDetailsListForStudent(account.googleId);
-    
-            data.sessionSubmissionStatusMap = generateFeedbackSessionSubmissionStatusMap(data.courses, account.googleId);
+            Map<String, StudentAttributes> courseToStudentAttributesMap = new HashMap<String, StudentAttributes>();
```

Comment:
Is this used? It is not clear what it contains as well. (courseId -> StudentAttributes) means one courseId can only map to one student?